rocm_docs/tensorflow-install-basic.md

```diff
@@ -61,12 +61,11 @@ Install ROCm pkgs:
 ```
 sudo apt-get update && \
     sudo apt-get install -y --allow-unauthenticated \
-    rocm-dkms rocm-dev rocm-libs hipcub rccl \
+    rocm-dkms rocm-dev rocm-libs rccl \
     rocm-device-libs \
     hsa-ext-rocr-dev hsakmt-roct-dev hsa-rocr-dev \
     rocm-opencl rocm-opencl-dev \
-    rocm-utils \
-    miopen-hip miopengemm
+    rocm-utils 
```

Comment:
removing `hipcub`, `miopen-hip` and `miopengemm` from the install list...is that by design?

Reply:
checked with Peng...the changes are intended as those packages are now part of the `rocm-libs` meta-package